### DISCUSSION TITLE
Similar to Jazzband?

### DISCUSSION THREAD

**Discussion by sebastian-philipp** [django-commons/membership#54] [General]
Feels to me as if this is really similar to https://github.com/jazzband . Does it make sense to join forces with them?

**Comment by tim-schilling** [comment]
Hi @sebastian-philipp. The decision to create Django Commons took a long time. There were months of preparation and organizing. We're currently at the point that we are interested in seeing how far we can take Django Commons. That's not to say things won't change in the future, but for now I think they'll coexist.

Jannis also had a response from the Jazzband side [here](https://publicidentity.net/@jezdez/112486115174839612).

**Reply by sebastian-philipp** [reply]
Thank you very much! Good luck to you and Jannis then!
